pilot/cmd/pilot-discovery/main.go

```diff
@@ -112,8 +112,12 @@ func init() {
 		"Consul Config file for discovery")
 	discoveryCmd.PersistentFlags().StringVar(&serverArgs.Service.Consul.ServerURL, "consulserverURL", "",
 		"URL for the Consul server")
+	discoveryCmd.PersistentFlags().DurationVar(&serverArgs.Service.Consul.Interval, "consulserverInterval", 2*time.Second,
+		"Interval for the Consul server")
```

Comment:
Can we make the comment a little more clear .... maybe something like "Interval (in seconds) for polling the Consul service registry"?

Reply:
@nmittler Thanks for your advice, I have fixed it.